indexing-service/src/test/java/io/druid/indexing/common/task/AppenderatorDriverRealtimeIndexTaskTest.java

```diff
@@ -424,13 +431,19 @@ public void testLateData() throws Exception
     awaitHandoffs();
 
     for (DataSegment publishedSegment : publishedSegments) {
-      Optional<Map.Entry<SegmentDescriptor, Pair<Executor, Runnable>>> optional = handOffCallbacks.entrySet().stream()
-                                                                                                  .filter(e -> e.getKey().equals(new SegmentDescriptor(
-                                                                                                      publishedSegment.getInterval(),
-                                                                                                      publishedSegment.getVersion(),
-                                                                                                      publishedSegment.getShardSpec().getPartitionNum()
-                                                                                                  )))
-                                                                                                  .findFirst();
+      Optional<Map.Entry<SegmentDescriptor, Pair<Executor, Runnable>>> optional = handOffCallbacks
```

Comment:
This construction looks like handOffCallbacks.get()...

Reply:
I'd rather not change any logic. This just retains the prior code